packages/webpack-extraction-plugin/src/parseCSSRules.ts

```diff
@@ -37,7 +37,7 @@ export function parseCSSRules(css: string) {
 
     if (cssBucketName) {
       const cssRule = serialize([element], stringify);
-      const bucketEntry: CSSBucketEntry = element.type === MEDIA ? [cssRule, cssMeta!] : cssRule;
```

Comment:
This condition was not safe as only rules from `m` bucket have `cssMeta` information.